scanners/subdomains.py

```diff
@@ -141,25 +140,22 @@ def scan(domain, options):
         logging.debug("\tSkipping, really down somehow, status code 0 for all.")
         return None
 
-    
     # bad hostname for cert?
-    if (protocol == "https") and (endpoint.get("https_bad_name", False) == True):
-        bad_cert_name = True
+    if (protocol == "https") and (endpoint.get("https_bad_name", False) is True):
+        bad_cert_name = True  # nopep8
     else:
-        bad_cert_name = False
-
-
+        bad_cert_name = False  # nopep8
```

Comment:
pep8 is complaining that this variable is never used. I'm not sure why this code is here so I silenced pep8